copulas/univariate/KDEUnivariate.py

```diff
@@ -0,0 +1,39 @@
+import logging
```

Comment:
This comment doesn't belong to this concrete line, but to the whole module: There is not a single unittest.

There should be at least one single test for each method, making it run in controlled, standard conditions and checking the behavior is the expected. There should be also tests covering corner cases, know issues and limitations of the functions.

In the case of this module, I'll start with an schema like this:
```python
from unittest import TestCase

from copulas.univariate.KDEUNivariate import KDEUNivariate


class TestKDEUnivariate(TestCase):

    def test___init__(self):
        """On init, param and column attributes are set to None."""
        pass

    def test_fit(self):
        """On fit, kde model is instantiated with passed data"""
        pass

    def test_fit_empty_data(self):
        """If fitting kde model with empty data it will crash""" # Will it?
        pass

    def test_get_pdf(self):
        """get_pdf evaluates with the model"""
        pass

    def test_get_pdf_invalid_value(self):
        """Evaluating an invalid value will crash"""
    ...
```